infra-templates/swarm/3/rancher-compose.yml

```diff
@@ -0,0 +1,24 @@
+.catalog:
+  version: v1.13-beta.1
+  minimum_rancher_version: 1.2.2
```

Comment:
Would we want to change the Rancher version to 1.4.0 or something later as Docker 1.13 is currently "unsupported"?